superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigModal.tsx

```diff
@@ -580,8 +601,26 @@ function FiltersConfigModal({
     <StyledModalWrapper
       visible={isOpen}
       maskClosable={false}
-      title={t('Add and edit filters')}
-      width="50%"
+      title={
+        <div
+          css={css`
+            display: flex;
+            justify-content: space-between;
+            align-items: flex-end;
+            margin-right: 50px;
+          `}
+        >
+          <div>{t('Add and edit filters')}</div>
+          <ToggleIcon
+            iconSize="l"
+            iconColor={theme.colors.grayscale.base}
+            onClick={toggleExpand}
+          />
+        </div>
+      }
+      width="70%"
+      {...(expanded && { width: '100%', height: '100%' })}
```

Comment:
Given that `StyleModalWrapper` is already receiving the `expanded` property, could you deal with `width` and `height` there and remove these two lines?